### PR TITLE
Rename nobulk -> snmp_bulk

### PR DESCRIPTION
**Pull request by Jellyfrog**
Rename option to align with the rest.
Please note it switches from a negative to positive option!

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by Jellyfrog** [comment]
Please note it switches from negative to positive option

**Comment by PipoCanaja** [comment]
Hi @Jellyfrog 
Shouldn't we add a notification to this change?

**Comment by Jellyfrog** [comment]
Doubt it's needed

File: includes/snmp.inc.php [hunk 1]
```diff
@@ -129,7 +129,7 @@ function gen_snmpget_cmd($device, $oids, $options = null, $mib = null, $mibdir =
  */
 function gen_snmpwalk_cmd($device, $oids, $options = null, $mib = null, $mibdir = null, $strIndexing = null)
 {
-    if ($device['snmpver'] == 'v1' || (isset($device['os']) && Config::getOsSetting($device['os'], 'nobulk'))) {
+    if ($device['snmpver'] == 'v1' || (isset($device['os']) && Config::getOsSetting($device['os'], 'snmp_walk', true) == false)) {
```
Comment (murrant) [inline includes/snmp.inc.php:132]: snmp_walk  --> snmp_bulk

**Review by murrant — CHANGES_REQUESTED** [review]
fix typo ;)

File: includes/snmp.inc.php (hunk 1 above)
Comment (Jellyfrog) [reply includes/snmp.inc.php:132]: Argh. How did I manage that...

**Comment by murrant** [comment]
We probably do need a notification or backwards compatibility.   This is a likely option for people to set in their local config for problem devices.

**Comment by Jellyfrog** [comment]
Hmm I see.

Well dont know if we should do it with the deprecated-config function thingy. because we need to loop all OSs...

Notification might be enough? :)

**Comment by murrant** [comment]
Yep, fine with just a notification.  You can add it to the in-tree rss or the one on the web site.

**Review by murrant — APPROVED** [review]

**Pull request merged**